File: spr2018_helpdesk-master/2125/NumericalAnalysis/RunNumericalAnalysis.java
public class RunNumericalAnalysis{
    public static void main(String[]  args){
        if(args.length != 2){
            System.out.println("Usage: java RunNumericalAnalysis low high");
        }
        double low = Double.parseDouble(args[0]);
        double high = Double.parseDouble(args[1]);
        NumericalAnalysis myTool = new NumericalAnalysis();
        double ans = myTool.solveForZero(new Square(-1), low, high);
        System.out.println("The answer is: " + ans);
    }
}

File: spr2018_helpdesk-master/2125/NumericalAnalysis/c++/RunNumericalAnalysis.cc
#include <stdlib.h>
#include <iostream>
#include <string>
#include "NumericalAnalysis.h"
using namespace std;
int main(int argc, char *argv[]){
    if(argc != 3){
        cout << "Usage: " << argv[0] << " low high" << endl;
        exit(EXIT_FAILURE);
    }
    double low = stod(argv[1]);
    double high = stod(argv[2]);
    NumericalAnalysis myTool;
    SomeFunction *func = new SomeFunction();
    double ans = myTool.solveForZero(func, low, high);
    cout << "The answer is: " << ans << endl;
}
